README.md

```diff
@@ -0,0 +1,65 @@
+<a href="#readme"><img src="https://tanker.io/images/github-logo.png" alt="Tanker logo" width="175" /></a>
+
+[![Actions status](https://github.com/TankerHQ/identity-python/workflows/tests/badge.svg)](https://github.com/TankerHQ/identity-python/actions)
+[![PyPi package](https://img.shields.io/pypi/v/tankersdk_identity.svg)](https://pypi.org/project/tankersdk_identity)
```

Comment:
Nitpick: redirects to `https://pypi.org/project/tankersdk-identity` with a `-`, do we want to target that URL directly?

Reply:
True, I didn't catch that! Fixed.